playwright/tests/screenshots/browsers.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { test, expect, type Page } from '@playwright/test';
+import { InscriptionView } from '../pageobjects/InscriptionView';
+import type { Section } from '../pageobjects/Section';
+import { screenshot } from './screenshot-util';
+import { Browser } from '../pageobjects/CodeEditor';
+
+const GENERIC_PID = {
+  SCRIPT: '168F0C6DF682858E-f3',
+  USER_TASK: '168F0C6DF682858E-f5',
+  DATABASE: '18141E75C9CEDD35-f3'
+} as const;
+
+test.describe('Browsers', () => {
+  test('ScriptingBrowser', async ({ page }) => {
+    await openSection(page, GENERIC_PID.SCRIPT, 'Output', 'Code').then(async section => {
+      const browser = await section.scriptArea().openBrowsers();
+      const dialog = browser.dialog();
```

Comment:
? 
do you like nested code?

why not 
```suggestion
    const section = await openSection(page, GENERIC_PID.SCRIPT, 'Output', 'Code');
    const browser = await section.scriptArea().openBrowsers();
    const dialog = browser.dialog();
```